nav-challenge/src/components/containers/signUpContainer/SignUpContainer.tsx

```diff
@@ -0,0 +1,106 @@
+import moment from "moment";
+import React, { FC, useState } from "react";
+import AppProvider, {
+  appContext,
+  useDispatchApp,
+} from "../../../store/AppProvider";
+import store from "../../../store/Index";
+import { fetchRegisterUser } from "../../../store/user/UserActions";
+import { SignUpFormProps } from "../../../utils/types/SignUpFormProps";
+import { UserSignUpDataType } from "../../../utils/types/UserSignUpDataType";
+import SignUpPage from "../../pages/signUpPage/SignUpPage";
+
+const SignUpProvider: FC = () => {
+  return (
+    <AppProvider store={store} context={appContext}>
```

Comment:
I believe that we should be using the `AppProvider` by wrapping the `App` component but I am not sure about the implementation details or the idea behind that provider